src/components/GoogleLoginButton.js

```diff
@@ -23,18 +19,20 @@ const GoogleLoginButton = () => {
         render={renderProps => (
           <Button
             type="primary"
+            style={{
+              background: '#333333',
+              border: 'none',
+              color: '#fff6ee',
+              fontFamily: 'Poppins',
```

Comment:
Select a standardized **font** to be used throughout the app and set it in `src/index.css`. Providing it everytime won't be a good option.

Reply:
Okay.